server/gateway.go

```diff
@@ -118,6 +118,14 @@ func (im GatewayInterestMode) String() string {
 	}
 }
 
+var gwDoNotForceInterestOnlyMode bool
+
+// GatewayDoNotForceInterestOnlyMode is used ONLY in tests.
+// DO NOT USE in normal code or if you embed the NATS Server.
+func GatewayDoNotForceInterestOnlyMode(doNotForce bool) {
```

Comment:
Should we make lower case to enforce this a bit more? Will work for tests under server but not test then..

Reply:
That's for that reason I have it exported, I need it in the test/ package.